### ISSUE TITLE
Copying the data from a report to the closure creation form no longer works

### ISSUE THREAD

**Issue by Glodenox** [Glodenox/wme-road-events#20]
For some reason the description and date are no longer filled in on the closure report form whenever the details of a report are displayed. Either add a button to copy the data once the details have been put on the right (issue #19) or fix the functionality.

**Comment by Hipska** [comment]
🤔 did that ever work?

**Comment by Glodenox** [comment]
Absolutely. Though it only worked for a couple of versions. I can't remember any more what change broke it.
